src/components/layers/Layer.js

```diff
@@ -523,31 +633,45 @@ export class UnconnectedLayer extends Component {
 		/*If layerType==latlon, then we display the elements to choose
            node labels and properties to be used.
          */
-		if (this.state.layerType !== LAYER_TYPE_LATLON)
+		const { rendering, layerType, nodes, nodeLabel, relationships, relationshipLabel,
```

Comment:
couldn't stop myself from doing this. destructuring  is such a great feature, wanted to use it everywhere but it would've introduced way too much unrelated changes